Show proper string when torrent availability is not available

Chocobo1 opened the pull request:
* Show proper string when torrent availability is not available
* Clean up coding style
* Apply "Hide zero values" to "Time Active" column
* Apply "Hide infinity values" to "Down/Up Limit" columns
* Apply "Hide infinity values" to ETA column

glassez reviewed (approved):
I hope the additional changes are not controversial... I don't use "Hide zero/infinity values" and don't have detailed views about it.

thalieht commented:
>Apply "Hide infinity values" to ETA column 

I'm totally in favor of this but i gotta mention that it was a conscious decision by several people when this feature was implemented.

thalieht commented on src/gui/transferlistmodel.cpp:
```diff
@@ -239,13 +239,16 @@ QString TransferListModel::displayValue(const BitTorrent::Torrent *torrent, cons
     {
-        return (hideValues && (value <= 0))
-                ? QString {} : Utils::String::fromDouble(value, 3);
+        if (hideValues && (value <= 0))
```
```suggestion
        if (hideValues && (value == 0))
```

there is no infinity in this column and this would still happen:
>because now i see the column start empty (zeroes), continue with the sort and then it becomes empty again (-1).

Chocobo1 commented:
>>Apply "Hide infinity values" to ETA column
>
>I'm totally in favor of this but i gotta mention that it was a conscious decision by several people when this feature was implemented.

I think we can try again and revert if there is still too much resistance.

PR updated, addressed all comments.
